Source/cielim/Private/RenderingFunctionsLibrary.h

```diff
@@ -22,21 +22,9 @@ class  CIELIM_API URenderingFunctionsLibrary :
 	UFUNCTION(BlueprintCallable, Category="Rendering Functions")
 	static void ApplyPSF_Gaussian();
 
-	UFUNCTION(BlueprintCallable, Category="Rendering Functions")
-	static void WriteImage();
-
 	UFUNCTION(BlueprintCallable, Category="Rendering Functions")
 	static void ApplyCosmicRays();
 
 	UFUNCTION(BlueprintCallable, Category="Rendering Functions")
-	static void ApplyReadNoise();
-
-	static void ReadImage(){};
-
-	
-
-	
-};
-
-
-
+	FString ApplyReadNoise(FString Filepath, float ReadNoiseSigma, float SystemGain);
+};
```

Comment:
Missing an extra line here (bottom of this file)

Reply:
Done! Pushed this change to the branch